### ISSUE TITLE
When I long press to select the text, the cursor cannot be dragged because it is blocked by the options.

### ISSUE THREAD

**Issue by drc5521** [kiwibrowser/src.next#190]
When I long press to select the text, the cursor cannot be dragged because it is blocked by the options.
As the picture shows.
![qq_pic_merged_1627260700897](https://user-images.githubusercontent.com/55195642/126919480-a7872801-96e9-4428-9992-5cc529ebe936.jpg)

**Comment by Lartsch** [comment]
Yes, this occurs very often for me as well (not always, but I cant make out what triggers ist).
I would consider this as a high priority bug @d3ward

**Comment by Lartsch** [comment]
Duplicate of #170 . Please close @drc5521

**Comment by kiwibrowser** [comment]
Fixed

**Comment by drc5521** [comment]
> Fixed

In the version of the generation 1087236204, the magnifying glass has a problem. When you choose, the magnifying glass is just under your finger, you can't see the zoom

**Comment by Lartsch** [comment]
I can confirm what @drc5521 says, also have the magnifier issue in the lastest build. Moving the cursors is possible yet not perfectly convenient with the magnifier covering parts of it.
